games/godot/Portfile

```diff
@@ -22,24 +22,52 @@ long_description    Godot Engine is a cross-platform game engine for \
                     (Linux, macOS, Windows), as well as mobile \
                     (Android, iOS) and web-based (HTML5) platforms.
 
-checksums           rmd160  4ff4f9c4798d67cf6cf4d63c38bed84d400daa8e \
+if {$subport eq ${name}} {
+    github.setup    godotengine ${name} 3.3 "" -stable
+    version         3.3.0
+
+    checksums       rmd160  4719f38a85cf339fa6c66d5849b972cd22c0a1ed \
+                    sha256  a65e296c900f442e19600853896f8f742a1414cc8864f9f6e45eeabc9c4b3362 \
+                    size    29432959
+
+    compiler.blacklist-append {clang < 802}
```

Comment:
please add a comment about why this is needed? If a certain compiler standard is required that should ideally be specified instead of blacklisting "manually"

Reply:
Good suggestion, but I'm not really sure what sort of comment I should add? My blacklisting isn't because a certain compiler standard is required; it's because Xcode Clang fails to compile the Godot source code for versions of Xcode older than 802. The blacklisting simply forces machines with Xcode older than 802 to use MacPorts Clang instead of attempting to use Xcode Clang. I'm not really sure how to express all of that succinctly in a comment. Any suggestions?

Reply:
Added comment stating that port fails to build using Xcode older than 802